cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/S3TestConfigSourceRel.java

```diff
@@ -0,0 +1,68 @@
+package com.yugabyte.cdcsdk.testing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.yugabyte.cdcsdk.server.TestConfigSource;
+
+public class S3TestConfigSourceRel extends TestConfigSource {
+    public S3TestConfigSourceRel() {
+        Map<String, String> s3Test = new HashMap<>();
+        String dbStreamId = "";
+        System.out.println("Going to create a stream ID");
+        try {
+            dbStreamId = TestHelper.getNewDbStreamId("yugabyte");
+        }
+        catch (Exception e) {
+            System.out.println("Exception thrown while creating stream ID: " + e);
+        }
+        System.out.println("Created stream ID: " + dbStreamId);
+
+        s3Test.put("cdcsdk.sink.type", "s3");
+        s3Test.put("cdcsdk.sink.s3.bucket.name", "cdcsdk-test-vk");
+        s3Test.put("cdcsdk.sink.s3.region", "us-west-2");
+        s3Test.put("cdcsdk.sink.s3.basedir", "S3ConsumerIT/");
+        s3Test.put("cdcsdk.sink.s3.pattern", "stream_{EPOCH}");
+        s3Test.put("cdcsdk.sink.s3.flushRecords", "3");
+        s3Test.put("cdcsdk.server.transforms", "FLATTEN");
+
+        s3Test.put("cdcsdk.source.connector.class", "io.debezium.connector.yugabytedb.YugabyteDBConnector");
+        // s3Test.put("cdcsdk.source." + StandaloneConfig.OFFSET_STORAGE_FILE_FILENAME_CONFIG,
```

Comment:
OFFSET STORAGE should not be used. I used it in S3ConsumerIT because I was using Postgres as source.